src/types/index.ts

```diff
@@ -74,7 +74,12 @@ export type PreviewResponse = {
   rows: ResultRow[];
 };
 
-export type Response = LoadResponse | PreviewResponse;
+export type CheckNullAidsResponse = {
+  columns: ResultColumn[];
+  rows: ResultRow[];
```

Comment:
This amount of data seems overkill for this call. I think a simple boolean field is enough here.

Reply:
hm, this is what we're returning from the `SELECT...` query underneath. I think that's beyond our control, or maybe I'm missing sth?

We're deriving the boolean result on a later stage of processing the JSON response.

Reply:
But we don't have to return the output of the query to the GUI.
The service knows the semantics of the query, it should also be the one interpreting the result.
If, in the future, the check becomes more complicated, only the service has to be updated.